app/channels/application_cable/connection.rb

```diff
@@ -1,4 +1,39 @@
 module ApplicationCable
   class Connection < ActionCable::Connection::Base
+    identified_by :current_user, :onsite
+
+    def connect
+      # Identify if the user is 'onsite' so that pre-rendered html can be sent
+      self.current_user, self.onsite = find_verified_user
+      log_tag = current_user.try(:name) || SecureRandom.uuid
+      logger.add_tags('ActionCable', log_tag)
+    end
+
+    def oauth_token
+      @oauth_token
+    end
+
+    def oauth_token=(token)
+      @oauth_token = token
+    end
+
+    protected
+
+    def find_verified_user
+      onsite = ['splits.io', 'localhost'].include?(request.host)
```

Comment:
You were right it didn't work for a beta, but I think I found a better solution.  `onsite = ['splits.io', 'localhost'].include?(request.domain(1))`.  This will take the first 2 items from the right, so the `splits.io` part, or if there isn't that many the entire thing (`localhost`).  This way it doesn't matter if we ever change the subdomain or add another or something like that.